Assets/Countly/Plugins/CountlySDK/Services/UserDetailsCountlyService.cs

```diff
@@ -476,13 +476,14 @@ private void AddToCustomData(string key, object value)
         /// <param name="key">property name</param>
         private bool ValidateRequest(string key)
         {
+            string caller = new System.Diagnostics.StackTrace().GetFrame(1).GetMethod().Name;
             if (!_consentService.CheckConsentInternal(Consents.Users)) {
-                Log.Debug("[UserDetailsCountlyService] ValidateRequest, Consent is not given, ignoring the request.");
+                Log.Debug($"[UserDetailsCountlyService] {caller}, Consent is not given, ignoring the request.");
```

Comment:
well that is also an option, I thought adding a new function parameter and just passing it

Reply:
It is a good viewpoint, I dont know can it cause anything but seems ok

Reply:
PS: please also fix codacy if it is making sense

Reply:
Technically it can not create issues since it'll always be called from another method, adding the method as a param and passing seemed like an unnecessary duplication to me

Reply:
Sure then, lets cook